include/jwt.h

```diff
@@ -73,7 +98,7 @@ typedef enum jwt_alg {
  *     success.
  * @return 0 on success, valid errno otherwise.
  */
-int jwt_new(jwt_t **jwt);
+JWT_EXPORT int jwt_new(jwt_t **jwt);
```

Comment:
Before I pull this, I need to make sure none of these ifdef's mess up the doxygen generation. I'll verify that.